Description: Alter class numbers in bro_ftp and bro_syslog
 .
 securityonion-elsa-extras (20131117-1ubuntu0securityonion3) precise; urgency=low
 .
   * Initial release
Author: Scott Allen Runnels <srunnels@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- securityonion-elsa-extras-20131117.orig/contrib/parsers/bro_ftp
+++ securityonion-elsa-extras-20131117/contrib/parsers/bro_ftp
@@ -1,7 +1,7 @@
-<ruleset name="bro_ftp" id='26001'>
+<ruleset name="bro_ftp" id='26003'>
   <pattern>bro_ftp</pattern>
   <rules>
-    <rule provider="Security_Onion" class='26001' id=''>
+    <rule provider="Security_Onion" class='26003' id=''>
       <patterns>
         <pattern>@ESTRING::|@@ESTRING::|@@ESTRING:i0:|@@ESTRING:i1:|@@ESTRING:i2:|@@ESTRING:i3:|@@ESTRING::|@@ESTRING::|@@ESTRING:s0:|@@ESTRING:s1:|@@ESTRING:s2:|@@ESTRING::|@@ESTRING:i4:|@@ESTRING::|@@ESTRING:s3:|@@ESTRING::|@@ESTRING::@</pattern>
       </patterns>
--- securityonion-elsa-extras-20131117.orig/contrib/parsers/bro_syslog
+++ securityonion-elsa-extras-20131117/contrib/parsers/bro_syslog
@@ -1,7 +1,7 @@
-<ruleset name="bro_syslog" id='26003'>
+<ruleset name="bro_syslog" id='26001'>
   <pattern>bro_syslog</pattern>
   <rules>
-    <rule provider="Security_Onion" class='26003' id=''>
+    <rule provider="Security_Onion" class='26001' id=''>
       <patterns>
         <pattern>@ESTRING::|@@ESTRING::|@@ESTRING:i0:|@@ESTRING:i1:|@@ESTRING:i2:|@@ESTRING:i3:|@@ESTRING::|@@ESTRING:s0:|@@ESTRING:s1:|@@ESTRING:s2:@</pattern>
       </patterns>
